serverlessworkflow/modules/ROOT/pages/index.adoc

```diff
@@ -383,3 +383,14 @@ xref:integrations/expose-metrics-to-prometheus.adoc[Exposing workflow base metri
 [.card-description]
 Exposing the workflow base metrics to Prometheus
 --
```

Comment:
Maybe here move `Use Cases` between `Cloud` and `Integrations` so the order is the same as in the nav panel?

Reply:
+1